artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/jgroups/JChannelManager.java

```diff
@@ -34,7 +34,7 @@
 
    private static final Logger logger = Logger.getLogger(JChannelManager.class);
 
-   private Map<String, JChannelWrapper> channels;
```

Comment:
@lulf that doesn't explain (at least I don't think so) why you had to make channels a static attribute